spec/index.html

```diff
@@ -648,27 +648,26 @@ <h3>IRIs</h3>
         Interoperability problems can be avoided by minting
         only IRIs that are normalized according to
         <a data-cite="rfc3987#section-5">Section 5</a>
-        of [[!RFC3987]]. Non-normalized forms that are best avoided
-        include the following:</p>
+        of [[!RFC3987]]. 
+        </p>
 
       <ul>
-        <li>Uppercase characters in scheme names and domain names</li>
-        <li>Percent-encoding of characters where it is not
-          required by IRI syntax</li>
-        <li>Explicit inclusion of the HTTP default port
-          (<code>http://example.com:80/</code>);
-          <code>http://example.com/</code> is preferable</li>
-        <li>A completely empty path in an HTTP IRI
-          (<code>http://example.com</code>);
-          <code>http://example.com/</code> is preferable</li>
-        <li>“<code>/./</code>” or “<code>/../</code>” in the path
-          component of an IRI</li>
-        <li>Lowercase hexadecimal letters within percent-encoding
-          triplets (i.e., “<code>%3F</code>” is preferred over
-          “<code>%3f</code>”)</li>
-        <li>Punycode-encoding of Internationalized Domain Names
-          in IRIs [[RFC3492]]</li>
-        <li>IRIs that are not in Unicode [=Normalization Form C=] [[I18N-Glossary]]</li>
+        <li>Use lowercase characters in scheme names and domain names.</li>
+        <li>Only use percent-encoding of characters where 
+          necessary for IRI syntax.</li>
```

Comment:
Probably should should be slightly different:

```suggestion
        <li>Only use percent-encoding of characters where
          required by the IRI syntax.</li>
```

Reply:
Done in #155.